Makefile

```diff
@@ -5,10 +5,14 @@
 export IMAGE_REPO ?= quay.io/operator-framework/operator-controller
 export IMAGE_TAG ?= devel
 export GO_BUILD_TAGS ?= upstream
+
+# If any of these versions are modified, remember to run `make ensure-external-manifests` to update 
```

Comment:
Do we have a target like `make verify` that runs things like generators, formaters, etc. and then runs `git diff -e` to check that nothing changed?

If so, it seems like we should add `ensure-external-manifests` to the list?

If not, capture a follow-up to add a `verify` target and run it in CI?